packages/platform-server/src/utils.ts

```diff
@@ -55,10 +68,12 @@ function appendServerContextInfo(applicationRef: ApplicationRef) {
 async function _render(platformRef: PlatformRef, applicationRef: ApplicationRef): Promise<string> {
   const environmentInjector = applicationRef.injector;
 
+  const platformState = platformRef.injector.get(PlatformState);
+  appendMarkerComment(platformState.getDocument());
```

Comment:
Just a proposal for a function name :) (to mimic the proposed const name)
```suggestion
  const platformState = platformRef.injector.get(PlatformState);
  appendSsrContentIntegrityMarker(platformState.getDocument());
```